packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -0,0 +1,713 @@
+  ///
+  /// Defaults to [false]. Must not be null. Cannot be true if [clipToSize]
+  /// is also true since children outside the viewport will be clipped, leading
+  /// to gratuitous waste.
+  final bool renderChildrenOutsideViewport;
+
+  /// List of children to scroll on top of the cylinder.
+  final List<Widget> children;
+
+  @override
+  Widget build(BuildContext context) {
+    return new Scrollable(
+      controller: controller,
+      physics: physics,
+      viewportBuilder: (BuildContext context, ViewportOffset offset) {
+        return new _ListWheelViewport(
+          diameterRatio: diameterRatio,
+          perspective: perspective,
+          itemExtent: itemExtent,
+          clipToSize: clipToSize,
+          renderChildrenOutsideViewport: renderChildrenOutsideViewport,
+          offset: offset,
+          children: children,
+        );
+      },
+    );
+  }
+}
+
+class _ListWheelViewport extends MultiChildRenderObjectWidget {
+  _ListWheelViewport({
+    Key key,
+    this.diameterRatio,
+    this.perspective,
+    this.itemExtent,
+    this.clipToSize,
+    this.renderChildrenOutsideViewport,
+    this.offset,
+    List<Widget> children,
+  }) : super(key: key, children: children);
+
+  final double diameterRatio;
+  final double perspective;
+  final double itemExtent;
+  final bool clipToSize;
+  final bool renderChildrenOutsideViewport;
+  final ViewportOffset offset;
+
+  @override
+  RenderListWheelViewport createRenderObject(BuildContext context) {
+    return new RenderListWheelViewport(
+      diameterRatio: diameterRatio,
+      perspective: perspective,
+      itemExtent: itemExtent,
+      clipToSize: clipToSize,
+      renderChildrenOutsideViewport: renderChildrenOutsideViewport,
+      offset: offset,
+    );
+  }
+
+  @override
+  void updateRenderObject(BuildContext context, RenderListWheelViewport renderObject) {
+    renderObject
+      ..diameterRatio = diameterRatio
+      ..perspective = perspective
+      ..itemExtent = itemExtent
+      ..clipToSize = clipToSize
+      ..renderChildrenOutsideViewport = renderChildrenOutsideViewport
+      ..offset = offset;
+  }
+}
+
+typedef double _ChildSizingFunction(RenderBox child);
+
+class _ListWheelParentData extends ContainerBoxParentData<RenderBox> { }
+
+/// Render, onto a wheel, a bigger sequential set of objects inside this viewport.
+///
+/// Takes a scrollable set of fixed sized [RenderBox]es and renders them
+/// sequentially along the scrolling axis.
+// Can be augmented later to have configurable axis directions.
+///
+/// It also starts with the first scrollable item in the center of the main axis
+/// and ends with the last scrollable item in the center of the main axis. This
+/// is in contrast to typical lists that starts with the first scrollable item
+/// at the start of the main axis and ends with the last scrollable item at the
+/// end of the main axis.
+///
+/// Also, instead of rendering its children on a flat plane, it renders them
```

Comment:
as above, avoid the "also, and also" pattern

Reply:
Done